front/src/pages/PlayPage.tsx

```diff
@@ -92,7 +93,12 @@ function PlayPage() {
 
   const swipe = async (dir: Direction) => {
     if (canSwipe && currentIndex < exercises.length) {
+      const exercise = exercises[currentIndex];
+      const answeredAsScam = dir === 'left';
       await childRefs[currentIndex].current?.swipe(dir); // Swipe the card!
+      if (exercise.scam === answeredAsScam) {
+        updateUserScore(score + 1);
+      }
```

Comment:
I feel like you are only updating the score if the user find that the card is a scam... But if the card isn't a scam and the user says it's not, he should also have its score increased !

Reply:
Ah no I just understood, it's working fine. I would just increase the score by 10 instead of 1 so that it's more satisfying for the user I think